### PR TITLE
Add support for --public-fqdn

### LINKED ISSUE #95: Feature Request: enable conditional --public-fqdn parameter
### Feature request

It would be great, if the command took into account the --public-fqdn parameter, while logging into the cluster. As stated in the Azure CLI documentation:

_--public-fqdn
Get private cluster credential with server address to be public fqdn.
default value: False_

It could be just a bool in the actions yaml to enable this :)

### PR DESCRIPTION
**Pull request by sblackstone**
Adds support for public-fqdn which is necessary for some private repositories.

Fixes #95

### CODE REVIEW

**Comment by sblackstone** [comment]
@microsoft-github-policy-service agree

**Comment by sblackstone** [comment]
Any chance this could get a review?  It's really straightforward....

**Review by davidgamero — APPROVED** [review]
Lgtm thanks for the PR!

**Comment by piroinno** [comment]
Can this PR be approved

File: action.yml [hunk 1]
```diff
@@ -19,6 +19,10 @@ inputs:
       description: 'Enables kubelogin for non-admin user scenario. Values: true or false'
       default: false
       required: false
+   public-fqdn:
+      description: 'Get private cluster credential with server address to be public fqdn.  Vaues: true or false'
```
Comment (OliverMKing) [inline action.yml:23]: nit: vaues typo

**Review by OliverMKing — APPROVED** [review]

File: action.yml (hunk 1 above)
Comment (sblackstone) [reply action.yml:23]: fixed.

**Comment by sblackstone** [comment]
I pushed a fix for the one issue that was mentioned, needs to be re-approved.

**Comment by sblackstone** [comment]
Can this please be merged.

**Comment by MattHodge** [comment]
@OliverMKing looks like this just needs your stamp of approval

**Comment by sblackstone** [comment]
Could we please merge this.   It's been 6 months - I fixed the single issue that came up in review.

**Comment by sblackstone** [comment]
Could we please merge this. It's been 9months - I fixed the single issue that came up in review.

**Review by davidgamero — APPROVED** [review]
lgtm

**Review by OliverMKing — APPROVED** [review]

**Pull request merged**